README.md

```diff
@@ -34,7 +34,7 @@ GOOGLE_APPLICATION_CREDENTIALS="/path/to/credentials.json" PROJECT_ID="my_projec
 
 #### Using Docker
 ```
-docker run -p 9411:9411 b.gcr.io/stackdriver-trace-docker/zipkin-collector
+docker run -v /path/to_credentials:/opt/gcloud -e GOOGLE_APPLICATION_CREDENTIALS="/opt/gcloud/credentials.json" -e PROJECT_ID="my_project_id" -p 9411:9411 b.gcr.io/stackdriver-trace-docker/zipkin-collector
```

Comment:
I think this new example will be good to have, but users on GCE could still use the existing example. Do you think it would make sense to move this example underneath the "Running on GCE" and "Using an explicit projectId and credentials file path" sections? Each of those sections could have an example running the JAR and an example using docker.